### ISSUE TITLE
[BUG] Blueprints loading is stuck in MacOS

### ISSUE THREAD

**Issue by eturino** [cabarius/ToyBox#126]
I'm using ToyBox both in Windows and MacOS. No problem at all with Windows, but when using MacOS the blueprints hang at 16.67%, and the Search n' Pick is basically frozen, returns no options.

The rest seems to be working fine.

## versions

- Game: `1.0.0u`; Mod: `1.2.10`; hangs at 16.67%
- Game: `1.0.0u`; Mod: `1.2.11`; hangs at 16.67%
- Game: `1.0.2b`; Mod: `1.2.11`; hangs at 16.66%
- Game: `1.0.2f`; Mod: `1.2.11`; hangs at 16.66%

**Comment by eturino** [comment]
I'm afraid that the log only says this:
```sh
❯ tail -f Log.txt
[Manager] Game version detected as '1.0.0'.
[Manager] Parsing mods.
[Manager] Reading file '/Users/eturino/Library/Application Support/Steam/steamapps/common/Pathfinder Second Adventure/Mods/0ToyBox0/Info.json'.
[Manager] Sorting mods.
[Manager] Loading mods.
[ToyBox] Version '1.2.10'. Loading.
[ToyBox] Active.
[Manager] FINISH. SUCCESSFUL LOADED 1/1 MODS.
[Manager] Spawning.
[Manager] Checking updates.
[ToyBox] calling BlueprintLoader.Load
```

and it stays there forever, no warning or error.

**Comment by cabarius** [comment]
This is unfortunate.  I'll have to setup on a mac and take a look.

**Comment by eturino** [comment]
we could have a shared-screen call if that helps. I'm afraid I don't know how to debug unity stuff, I've never worked with this kind of projects :(

**Comment by ThyWoof** [comment]
Thanks for reporting. Fixed and tested on MacOS 10.4.x
